frontend_lib/i18next.scanner/fr/translation.json

```diff
@@ -1,4 +1,5 @@
 {
+  "@all please notice that I did an important update on this content.": "@tous veuillez noter que j'ai fait une mise à jour importante sur ce contenu.",
```

Comment:
So we settled for translated mentions?

Reply:
yes, https://github.com/tracim/tracim/issues/2843 (third check point) we have just `@all` but it's translated